test/regtest.py

```diff
@@ -11,8 +11,12 @@
 
 # list of regression tests with the expected outputs
 tests = [
-  RegTest('hello',                 r'1 verified, 0 errors?', r'Program has no bugs', r'Program has no bugs', 2),
-  RegTest('hello_fail',            r'0 verified, 1 errors?', r'This assertion can fail', r'This assertion can fail', 2),
+  RegTest('jain_1_true',                r'1 verified, 0 errors?', r'Program has no bugs', r'Program has no bugs', 2),
```

Comment:
Every test should have a failing version as well. Also, sorry, but what does "jain" stand for?
Notice that our naming convention so far has been (maybe it is not the best one, but no point in switching at this point):
- test that should not report bugs: test_name.c
- test that should report bugs: test_name_fail.c
We do not use test_name_true.c. So you should rename those tests into just test_name.c. You should also create failing versions.